controllers/common/types.go

```diff
@@ -80,6 +80,7 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			r.Log.Error(err, "failed to recover chaos")
 			return ctrl.Result{Requeue: true}, nil
 		}
+		return ctrl.Result{}, nil
```

Comment:
This is a bug during handling the recover logic in `common chaos` controller, I fixed in this request when I found it.

Reply:
do we need `r.Update(ctx, chaos)` here?

Reply:
> do we need `r.Update(ctx, chaos)` here?

good catch. I adjust the common reconciler.